docs/guides/measure-qubits.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Measure qubits
+description: Learn how to measure qubits, including constraints on where measurements can be used.
+---
+
+# Measure qubits
+
+To get information about a qubit's state, you have to _measure_ it onto a [classical bit](/api/qiskit/circuit#qiskit.circuit.Clbit).  This is an irreversible operation, and usually destroys entanglement and phase coherence between the target qubit and the rest of the system. Thus, after a qubit is measured, you can't perform any more instructions on that qubit, and each qubit can only be measured once. Additionally, there must be at least one classical register to store the measurement information.
```

Comment:
Is the goal of this docs page to explain measurements themselves or how they can be used in Qiskit? The former is definitely more involved than the latter. Maybe it would be good to restrict to just explaining how they work in Qiskit? 🙂 

Some notes as to why explaining measurements is more involved:
* There are other ways of obtaining information about qubit states aside from directly measuring it (see e.g. "weak measurement" if you're interested), to be technically correct maybe "can" is a better word 🙂 
* Measurements don't need to be destructive (i.e. we don't know the qubit state after) -- in fact many quantum computing stacks (incl. IBMs) assume that it is projective, meaning that the qubit is projected onto the outcome we observed. 

How about reformulating as the following?
```suggestion
To get information about a qubit's state, you can _measure_ it onto a [classical bit](/api/qiskit/circuit#qiskit.circuit.Clbit). In Qiskit, measurements are performed in the computational basis, i.e., the single-qubit Pauli-$Z$ basis. A measurement will thus yield 0 or 1 depending on the overlap with the Pauli-$Z$ eigenstates $|0\rangle$ and $|1\rangle$.
```

I don't know how to apply math correctly, but we could add this equation to be even more explicit
<img width="400" alt="image" src="https://github.com/user-attachments/assets/e353affa-f6ad-42ef-8023-7526c0de0849">